model/src/form/form-definition/index.ts

```diff
@@ -12,14 +12,17 @@ import {
   type RelativeDateValueData
 } from '~/src/conditions/types.js'
 import {
+  type Repeat,
   type ConditionWrapper,
   type FormDefinition,
   type Item,
   type Link,
   type List,
   type Page,
   type PhaseBanner,
-  type Section
+  type Section,
+  type RepeatSchema,
+  type RepeatOptions
```

Comment:
Sorry, another one, see if you can find the ESLint option to do this automatically

Not sure how but my editor seems to do this for me in A-Z order (when auto adding imports)

```suggestion
  type Section,
  type Repeat,
  type RepeatSchema,
  type RepeatOptions
```